src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -232,27 +248,28 @@ public function placeAction(Request $request)
      * @return Response
      */
     #[AdminSecurity("is_granted('create', request.get('_legacy_controller'))")]
-    public function createAction(Request $request)
-    {
-        /** @var ShopContext $shopContextChecker */
-        $shopContextChecker = $this->get('prestashop.adapter.shop.context');
-
+    public function createAction(
+        Request $request,
+        ShopContext $shopContextChecker,
```

Comment:
I think we should remove the usage of this service and instead rely on `$isSingleShopContext = $this->getShopContext()->getShopConstraint()->getShopId() !== null` like in the other actions